docs/source/tutorial/interfaces.rst

```diff
@@ -282,7 +282,8 @@ applied:
 
 - ``x <- v; e`` becomes ``v >>= (\x => e)``
 
-- ``v; e`` becomes ``v >>= (\_ => e)``
+- ``v; e`` becomes ``v >> e``, where ``>>`` is the non-binding sequencing
+  operator defined as ``v >>= \_ => e``.
```

Comment:
This definition is correct only for `>>` defined for `Monad`, but is not correct in the general case. Since `do`-notation works syntactically, not only for `Monad`, this should be reformulated.

Reply:
Updated with a line above defining `>>`. Wasn't sure whether to include the type signature, or whether that over complicates the explanation, given it's now defined as `(>>) : Monad m => m () -> Lazy (m b) -> m b`.